web/models.py

```
@@ -2026,58 +2045,48 @@
-        return self.questions.count()
-
-    @property
-    def completion_count(self):
-        return self.user_quizzes.filter(completed=True).count()
-
 
 class QuizQuestion(models.Model):
     """Model for storing quiz questions."""
 
-    QUESTION_TYPES = [("multiple", "Multiple Choice"), ("true_false", "True/False"), ("short", "Short Answer")]
+    QUESTION_TYPES: ClassVar[List[Tuple[str, str]]] = [
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)